website/static/js/filepage/revisions.js

```diff
@@ -86,12 +90,31 @@ var FileRevisionsTable = {
             });
         };
 
+        self.showModal = function(hashName, index, hash) {
+            var showModal = bootbox.dialog({
+                title: 'Version ' + index + ' ' + hashName + ' hash',
+                message: '<div class="row">  ' +
+                    '<div class="col-md-12">' +
+                    '<div class="control-group">' +'<p class="col-xs-10">' + hash + '</p>' +
+                    '<a class="btn-sm btn-primary col-xs-2" style="align: right; text-align: center; ' +
+                    'padding: 5px; width: 80px" id="copyBtn" data-clipboard-text="' +
+                    hash + '"> Copy </a>'+
+                    '</div></div></div>'
+            });
+            showModal.on('show.bs.modal', function() {
+                var $copyBtn = $('#copyBtn');
+                new makeClient($copyBtn);
```

Comment:
This will create a new client every time one of the modals is shown, but the clients aren't destroyed when the modal is closed. Either (1) don't create a new client on every modal shown or (2) make sure to clean up the clients after closing the modal.